docs/conf.py

```diff
@@ -35,7 +35,7 @@
 import sphinx.environment
 from docutils.utils import get_source_line
 
-def _warn_node(self, msg, node):
+def _warn_node(self, msg, node, **kwargs):
     if not msg.startswith('nonlocal image URI found:'):
         self._warnfunc(msg, '%s:%s' % get_source_line(node))
 
```

Comment:
From what I see [here](https://travis-ci.org/crossbario/txaio/jobs/164390116) it fails because of it uses pytest3 but the test suite cannot yet be run under pytest3. See #78 for the appropriate fix. Once this is merged, I can rebase this PR to make the tests pass.